src/document-parser.ts

```diff
@@ -0,0 +1,61 @@
+import {DocumentDescriptor} from 'hydrolysis';
```

Comment:
This was grabbed out of analyzer (see above). Was it originally from some external source?